client/src/main.css

```
@@ -1,3 +1,3 @@
-@import "tailwindcss";
+@import 'tailwindcss';
 @import '@ucdavis/gunrock-tailwind/imports.css';
 @plugin "daisyui";
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@plugin`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)